spec/definitions/mixin_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
+
+RSpec.describe 'index mixin-ed class' do
+  let(:class_name) { 'Man' }
+  let(:include_module) { 'UpOnTheSky' }
+  let(:extend_module) { 'UnderTheSea' }
+  let(:prepend_module) { 'OnTheGround' }
+  let(:code) do
+    <<-CODE
+      class Man
+        include UpOnTheSky
```

Comment:
This test is simple. Let's add some more tests that includes multiple prepends and includes; as well as normal methods to test out the behaviors.